### PR TITLE
Update DG

### LINKED ISSUE #67: Edit Developer Guide to add Target User Profile, Value Proposition and Use Cases for corresponding assigned features

### PR DESCRIPTION
**Pull request by andrechuakj**
Closes #67 
- Update target user profile
- Update value proposition
- Add use case for Create Member command
- Underline use case titles for better clarity
- Standardize full stops at end of line for use cases.

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -289,16 +289,34 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
 
 **Guarantees: MSS -> Specified member will be deleted.**
 
 **MSS**
 
-1. User requests to <ins>list all members</ins> (UC02)
-2. CCACommander shows a list of all members
-3. User requests to delete a specific member in the list of all members
-4. CCACommander deletes the specific member
+1. User requests to <ins>list all members</ins> (UC02).
```
Comment (SelwynAng) [inline docs/DeveloperGuide.md]: This 'list all members' use case will be UC03, due to a change in ordering as the 'create member' use case is added.

**Review by SelwynAng — COMMENTED** [review]
LGTM aside from the minor change needed, which is stated in the review!

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (andrechuakj) [reply docs/DeveloperGuide.md]: Updated

**Comment by SelwynAng** [comment]
LGTM, thanks for making that minor change!

**Pull request merged**